tests/test_transit_coverage.py

```diff
@@ -1,17 +1,40 @@
 import numpy as np
+from exovetter import transit_coverage
 import pytest
 from numpy.testing import assert_allclose
 
-from exovetter import transit_coverage
```

Comment:
Usually the import order goes like this, so I don't think there is a reason to move this import up.

* Standard library (comes with Python).
* Third-party (from dependencies).
* Local (from `exovetter` itself).

And within each category, I like to put the `import ...` first followed by `from ...` and for each of those, alphabetical. But that is just personal preference.